cmd/activity-monitor/main.go

```diff
@@ -151,14 +146,12 @@ func main() {
 			syslogU, err := url.Parse(c.GlobalString("jsonlog"))
 			if err != nil {
 				log.Fatalf("Could not parse Syslog URL: %s", err)
-				return
 			}
 
 			logger.SetEndpoint(syslogU.Scheme, syslogU.Host)
 			err = logger.Connect()
 			if err != nil {
 				log.Fatalf("Could not open remote syslog: %s", err)
-				return
```

Comment:
@bifurcation See the docs:
> // Fatalf is equivalent to Printf() followed by a call to os.Exit(1).

So I am very confused as to why would the code keep executing.